R/tmb-model.R

```diff
@@ -6,7 +6,7 @@
 #'
 #' @seealso [select_naomi_data]
 #' @export
-prepare_tmb_inputs <- function(naomi_data) {
+prepare_tmb_inputs <- function(naomi_data, report_likelihood = 1L) {
```

Comment:
1. I suggest setting this to `0L` by default. I **think** that reporting this by default will mess up the outputs reporting functions. Is that correct?

2. Is there a reason putting this arg on function `prepare_tmb_inputs()` rathe than `make_tmb_obj()` where the `calc_outputs = ` argument is specified?

Reply:
Correction on #1. I just tested this -- I think that actually `sample_tmb()` and `output_package()` work just fine with the log-likelihood results output.  And also `sample_tmb()` will output the `_ll` for all posterior samples, which may be convenient for future applications. 

So maybe fine to leave this as 'yes' default. It will slightly increase the memory footprint of `$sample`, but that's maybe not such a big deal.

Reply:
Thanks, agreed on (1).

I had included it in `create_tmb_inputs()` so it can be specified without having to pull out the internal functions of `fit_tmb()` but it can be added in to `fit_tmb()` if you'd prefer to have the ability to change the option at that point?